website/docs/components/pagination/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,44 @@
+## Usage
+
+### When to use
+
+- To break down large content into pages. Usually paired with tables, but works with other types of page content.
+
+### When not to use
+
+- As a navigation control for a flow or to pair with a stepper, i.e. for a guide, tutorial, setup flow.
+- As a controller to switch between multiple views. Use [Tabs](/components/tabs/) instead.
+
+### Numbered vs Compact
+
+In most cases, the numbered pagination provides a greater user experience. It allows users to jump between pages and always return to the first page or go to the last page without navigating through the pages manually.
```

Comment:
```suggestion
In most cases, the numbered pagination provides a better user experience. It allows users to jump between pages and always return to the first page or go to the last page without navigating through the pages manually.
```

Reply:
I'm hesitant about this one because we refer to to a variant name, so I feel like it's correct in this case, but I'm not sure. I can't find a similar example on another component. @MelSumner @heatherlarsen do you have an opinion on this?